app/src/test/java/com/nervousfish/nervousfish/BaseTest.java

```diff
@@ -0,0 +1,99 @@
+package com.nervousfish.nervousfish;
```

Comment:
Yeah, the file is in develop already. Not sure why git sees this file a change compared to develop